source/amc_uploader/requirements.txt

```diff
@@ -1 +1,2 @@
+requests
```

Comment:
is this dependency added for a separate update? I notice it's not used in this PR

Reply:
Good catch. I removed it.